src/components/Search/Search.tsx

```diff
@@ -53,13 +54,29 @@ const Search: React.FunctionComponent = () => {
         </Col>
       </Row>
       <Row>
+      <Col></Col>
         <Col>
-          {!searchQuery
-            ? renderIntroduction()
-            : <SearchContent searchQuery={searchQuery} />
+
+        <Tabs defaultActiveKey="content" id="noanim-tab-example">
+          <Tab eventKey="people" title="People" peopleTab>
+          {!searchQuery || searchQuery === " "
+            ? (renderIntroduction())
+            : (<SearchPerson searchQuery={searchQuery} />)
+          }
+          </Tab>
+          <Tab  eventKey="content" title="Content" contentTab>
+          {!searchQuery || searchQuery === " "
+            ? (renderIntroduction())
+            : (<SearchContent searchQuery={searchQuery} />)
+          }
+          </Tab>
+          <Tab  eventKey="organisations" title="Organisations" disabled organisationsTab>
+          {!searchQuery || searchQuery === " "
+            ? (renderIntroduction())
+            : (<SearchPerson searchQuery={searchQuery} />)
```

Comment:
Neat that you've added this, but I think you meant SearchOrganizations here

Reply:
yeah, but that component doesn't exist yet, so I just left the other one. Should I still replace it?

Reply:
It does exist though